utils/maintenence.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+die() {
+    echo >&2 "$@"
+    exit 1
+}
+
+# check input
+CORE=${CORE_DIR?"Environment variable CORE_DIR must be set to the cyclus repository directory."}
+CYCA=${CYCAMORE_DIR?"Environment variable CYCAMORE_DIR must be set to the cycamore repository directory."}
+STUB=${STUB_DIR?"Environment variable STUB_DIR must be set to the cycstub repository directory."}
+[ "$#" -eq 1 ] || die "Input requires version, e.g., 
+$ ./maintence.sh 0.3.0
+" 
+VERSION=$1
+echo "Performing maintence updates for Cyclus stack verison $VERSION"
+
+# pyne
+git clone git@github.com:pyne/pyne
```

Comment:
You probably would be better off wget or curl -O the archive rather than cloning the repo....